catalog/app/containers/Auth/saga.js

```diff
@@ -150,6 +151,22 @@ function* signIn(credentials) {
   }
 }
 
+function* setBrowseCookie(tokens) {
+  try {
+    yield call(apiRequest, {
+      auth: { tokens, handleInvalidToken: false },
+      url: `${cfg.s3Proxy}/browse/set_browse_cookie`,
+      method: 'POST',
+      credentials: 'include',
+    })
+  } catch (e) {
```

Comment:
i guess we can just `log.warn` and `Sentry.captureException` it, since the error is never awaited anyways